specification/databox/resource-manager/Microsoft.DataBox/preview/2021-08-01-preview/examples/AvailableSkusPost.json

```diff
@@ -0,0 +1,280 @@
+{
+  "parameters": {
+    "subscriptionId": "fa68082f-8ff7-4a25-95c7-ce9da541242f",
+    "resourceGroupName": "bvttoolrg6",
+    "location": "westus",
+    "api-version": "2020-11-01",
```

Comment:
It seems the API version  of the examples are not correct. Please double check.

Reply:
made the corrections